docs/README/ja-jp.md

```diff
@@ -0,0 +1,112 @@
+<p align="right">
+  <a href="/README.md">
+  English
+  </a>
+  <span> | </span>
+  <a href="/docs/README/zh-cn.md">
+  简体中文
+  </a>
+  <span> | </span>
+  <a href="/docs/README/zh-tw.md">
+  正體中文
+  </a>
+  <span> | </span>
+  <strong>日本語</strong>
+  </a>
+</p>
+
+<h1 align="center">
+  <img src="https://github.com/Ehviewer-Overhauled/Art/blob/master/launcher_icon-web.svg" width="200" alt="EhViewer">
+  <br>EhViewer<br>
+</h1>
+
+<p align="center">
+  <a href="https://github.com/FooIbar/EhViewer/actions/workflows/ci.yml">
+    <img src="https://github.com/FooIbar/EhViewer/actions/workflows/ci.yml/badge.svg" alt="Github Actions">
+  </a>
+  <a href="/LICENSE">
+    <img src="https://img.shields.io/github/license/FooIbar/EhViewer" alt="LICENSE">
+  </a>
+  <a href="https://www.codefactor.io/repository/github/FooIbar/EhViewer">
+    <img src="https://www.codefactor.io/repository/github/FooIbar/EhViewer/badge" alt="CodeFactor">
+  </a>
+  <a href="https://github.com/FooIbar/EhViewer/releases">
+    <img src="https://img.shields.io/github/v/release/FooIbar/EhViewer" alt="Release">
+  </a>
+  <a href="https://github.com/FooIbar/EhViewer/issues">
+    <img src="https://img.shields.io/github/issues/FooIbar/EhViewer" alt="Issues">
+  </a>
+</p>
+
+<div align="center">
+  <h3>
+    <a href="#説明">
+    説明
+    </a>
+    <span> | </span>
+    <a href="#ダウンロード">
+    ダウンロード
+    </a>
+    <span> | </span>
+    <a href="#スクリーンショット">
+    スクリーンショット
+    </a>
+    <span> | </span>
+    <a href="#感謝">
+    感謝
+    </a>
+    <span> | </span>
+    <a href="#ライセンス">
+    ライセンス
+    </a>
+  </h3>
+</div>
+
+# 説明
+
+軽量で高性能な EhViewer のフォークです。
+
+[Material Design 3](https://m3.material.io/) と[ダイナミックカラー](https://m3.material.io/styles/color/dynamic-color/overview)のサポートを搭載しています。
+
+# ダウンロード
+
+| フレーバー    | 機能                            |
+|-------------|----------------------------------|
+| デフォルト    | Android 8.0 以降、完全対応       |
+| Marshmallow | Android 6.0-7.1、一部対応         |
```

Comment:
This was changed in https://github.com/FooIbar/EhViewer/commit/6fb20fc6c134e88c528a0ce59889c30e2eaaed7b, you may want to check that.
And "default" should not be translated.

Reply:
fixed.